Dear eLife Editorial Staff,

We are writing in regards to corrections to our accepted manuscript, "Simultaneous trimodal single-cell measurement of transcripts, epitopes, and chromatin accessibility using TEA-seq", DOI: 10.7554/eLife.63632, available at https://elifesciences.org/articles/63632.

After publication, we have discovered errors in the manuscript that we would like to correct.

The first such error is due to incorporation of two extra nucleotides, 'AT', in our i7 ADT primer sequences. We had corrected this in-house, and used the corrected version for all experiments, but did not correct the sequences in our manuscript.

This error appears in Supplementary file 5, "Custom oligonucleotide sequences used to generate ICICLE-seq libraries.", as well as in Figure 3 - figure supplement 1. We apologize for these errors, and have attached corrected versions of these files.


The second set of errors appear to stem from repetition of the last 3 digits in long numbers. These were not present in our submitted materials. We apologize for missing these corrections during the Proof stage:

In the Figure 1 - figure supplement 2 legend: ">1,000,000 unique fragments" should be ">1,000 unique fragments"

In the Results section, under the heading "Trimodal measurement of transcripts, epitopes, and accessibility with TEA-seq": ">2,500,500 unique ATAC-seq fragments" should be ">2,500 unique ATAC-seq fragments"; "median = 1,249,249 genes detected" should be "median = 1,249 genes detected".

In the Figure 4 legend: "n = 227,390,390 barcodes are displayed" should be "n = 227,390 barcodes are displayed"; "n = 34,757,757 total cells are displayed" should be "n = 34,757 total cells are displayed"; "those with >1,000,000 unique ATAC fragments" should be "those with >1,000 unique ATAC fragments"; "n = 7,939,939 barcodes" should be "n = 7,939 barcodes"

In Materials and methods, under heading "sn/scATAC-seq Quality Control": ">1,000,000 uniquely aligned fragments" should be replaced with ">1,000 uniquely aligned fragments".

In Materials and methods, under the heading "TEA-Seq analysis": ">2,500,500 unique fragments" should be ">2,500 unique fragments".

In Materials and methods, under the heading "Data analysis":"~20,000,000 reads per expected cell recovered" should be " ~20,000 reads per expected cell recovered"; "~48,00,800 reads per expected cell recovered" should be "~4,800 reads per expected cell recovered"; "TEA-seq: >500 genes, <2,750,750 genes" should be "TEA-seq: >500 genes, <2,750 genes"; "Multiome: >300 genes, <4,000,000 genes" should be "Multiome: >300 genes, <4,000 genes"; "Multiome: >300 genes, <2,750,750 genes" should be "Multiome: >300 genes, <2,750 genes"; "CITE-seq: >500 genes, <5,000,000 genes" should be "CITE-seq: >500 genes, <5,000 genes"; ">1,000,000 unique fragments" should be ">1,000 unique fragments"


Please let us know what the best course of action will be to update the manuscript with these corrections.

Sincerely,
Lucas Graybuck and Peter Skene
